### DISCUSSION TITLE
I just love this even more.

### DISCUSSION THREAD

**Discussion by hlepnamingblyat** [Monika-After-Story/MonikaModDev#9609] [QA - tech support + general] [answered]
I couldn't contain myself when Monika pouted. Sooo cute!! 💖 
But the new game isn't showing up... What am I missing here?

**Comment by Wingdinggaster656** [comment, marked as answer]
Check the `characters` folder lately?

**Comment by hlepnamingblyat** [comment]
> Check the `characters` folder lately

Sorry... it's late here... 😆
